bleachbit/Cleaner.py

```diff
@@ -134,6 +134,19 @@ def get_option_descriptions(self):
         if self.options:
             for key in sorted(self.options.keys()):
                 yield (self.options[key][0], self.options[key][1])
+                
+    def get_long_option_description(self, key):
+        """Yield the name and a potentially more verbose description for a single option"""
+        if self.options:
+            ret = self.options[key][1]
+            ret += "\n\nThis cleanup option will perform the following actions:\n\n"
```

Comment:
This string should be eligible for localization/translation. Wrap it in the `_()` function.